modular_skyrat/code/modules/cargo/bounties.dm

```diff
@@ -2,3 +2,8 @@
 	name = "Clarke"
 	reward = 20000
 	wanted_types = list(/obj/mecha/working/ripley/clarke)
+
+/datum/bounty/item/mech/killdozer
+	name = "Killdozer"
+	reward = 35000
+	wanted_types = list(/obj/mecha/working/ripley/firefighter/killdozer)
```

Comment:
undocumented change

Reply:
I'll make a separate PR for it. It should always have been there though.